Compilation errors with Angular 2.0.0-rc.2

Trying to compile `angular2-polymer` against the latest RC of Angular 2, we get following compilation errors.

```
src/polymer-element.ts(22,62): error TS2339: Property 'BrowserDomAdapter' does not exist on type '{ DomAdapter: typeof DomAdapter; getDOM: () => DomAdapter; setRootDomAdapter: (adapter: DomAdapte...'.
src/polymer-element.ts(25,10): error TS2339: Property 'appendChild' does not exist on type 'PolymerDomAdapter'.
src/polymer-element.ts(25,34): error TS2339: Property 'createTextNode' does not exist on type 'PolymerDomAdapter'.
src/polymer-element.ts(65,32): error TS2339: Property 'setDOM' does not exist on type '{ DomAdapter: typeof DomAdapter; getDOM: () => DomAdapter; setRootDomAdapter: (adapter: DomAdapte...'.
src/polymer-element.ts(68,32): error TS2339: Property 'setDOM' does not exist on type '{ DomAdapter: typeof DomAdapter; getDOM: () => DomAdapter; setRootDomAdapter: (adapter: DomAdapte...'.
```

--- a/src/polymer-element.ts
+++ b/src/polymer-element.ts
@@ -13,11 +13,12 @@ import {
 } from '@angular/core';
 import { NgControl, NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/common';
 
-import { __platform_browser_private__ } from '@angular/platform-browser';
+import { BrowserDomAdapter } from '@angular/platform-browser/src/browser/browser_adapter';
+import { setDOM } from '@angular/platform-browser/src/dom/dom_adapter';
 
 const Polymer:any = (<any>window).Polymer;
 
-class PolymerDomAdapter extends __platform_browser_private__.BrowserDomAdapter {
+class PolymerDomAdapter extends BrowserDomAdapter {
   createStyleElement(css:any, doc:Document = document) {
     var style:any = doc.createElement.call(doc, 'style', 'custom-style');
     this.appendChild(style, this.createTextNode(css));
@@ -60,9 +61,9 @@ class PolymerShadyDomAdapter extends PolymerDomAdapter {
 }
 
 if (Polymer.Settings.useShadow) {
-  __platform_browser_private__.setDOM(new PolymerDomAdapter());
+  setDOM(new PolymerDomAdapter());
 } else {
-  __platform_browser_private__.setDOM(new PolymerShadyDomAdapter());
+  setDOM(new PolymerShadyDomAdapter());
 }
 
 

